lib/services/authority.go

```diff
@@ -39,6 +39,15 @@ import (
 	"golang.org/x/crypto/ssh"
 )
 
+// AuthorityGetter defines interface for fetching cert authority resources.
+type AuthorityGetter interface {
```

Comment:
nit: seems there is already a `Trust` interface defined in trust.go, which is a super set of this. maybe either use that one, or move this over there and let `Trust` inherit this? Just personal preference to avoid defining same thing twice

Reply:
Moved to `services/trust.go` and embedded into the `Trust` interface.